standard/expressions.md

```diff
@@ -3700,6 +3700,7 @@ relational_expression
     | relational_expression '<=' shift_expression
     | relational_expression '>=' shift_expression
     | relational_expression 'is' type
+    | relational_expression 'is' pattern
```

Comment:
Meeting notes: there's ambiguity here. We should somehow reference how it's resolved. We *may* be able to merge these, but we'll need to talk about how nullable value types are okay for `type` but not `pattern`.

Reply:
Agreed, and also mention that the precedence of an expression, if the pattern is a constant pattern, is restricted to `shift_expression` or tighter.

Reply:
Though honestly it could be done later too.

Reply:
Added to #935